connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerSinkTaskTest.java

```diff
@@ -1951,6 +1960,27 @@ public void testOriginalTopicWithTopicMutatingTransformations() {
         PowerMock.verifyAll();
     }
 
+    @Test
+    public void testErrorReportersConfigured() {
+        RetryWithToleranceOperator retryWithToleranceOperator = createMock(RetryWithToleranceOperator.class);
+        createTask(initialState, keyConverter, valueConverter, headerConverter, retryWithToleranceOperator,
+                () -> Collections.singletonList(createMock(ErrorReporter.class)));
+
+        expectInitializeTask();
+        Capture<List<ErrorReporter>> errorReporters = EasyMock.newCapture();
+        retryWithToleranceOperator.reporters(EasyMock.capture(errorReporters));
+        PowerMock.expectLastCall();
+
+        PowerMock.replayAll(retryWithToleranceOperator);
+
+        workerTask.initialize(TASK_CONFIG);
+        workerTask.initializeAndStart();
+
+        assertEquals(1, errorReporters.getValue().size());
```

Comment:
Nit: we can get a little more granular about our assertions here by using `assertEquals` to compare collections instead of just an expected vs. actual collection's size.
```suggestion
        List<ErrorReporter> errorReporters = Collections.singletonList(createMock(ErrorReporter.class));
        createTask(initialState, keyConverter, valueConverter, headerConverter, retryWithToleranceOperator,
                () -> errorReporters);

        expectInitializeTask();
        Capture<List<ErrorReporter>> errorReportersCapture = EasyMock.newCapture();
        retryWithToleranceOperator.reporters(EasyMock.capture(errorReportersCapture));
        PowerMock.expectLastCall();

        PowerMock.replayAll(retryWithToleranceOperator);

        workerTask.initialize(TASK_CONFIG);
        workerTask.initializeAndStart();

        assertEquals(errorReporters, errorReportersCapture.getValue());
```

Reply:
Makes sense, done.